Makefile_i8_PaStiX

```diff
@@ -0,0 +1,131 @@
+# See our wiki for getting the CalculiX dependencies:
+# https://github.com/precice/calculix-adapter/wiki/Installation-instructions-for-CalculiX
+# Set the following variables before building:
+# Path to original CalculiX source (e.g. $(HOME)/ccx_2.16/src )
+CCX             = $(HOME)/CalculiX/ccx_2.17/src
+
+### Change these if you built SPOOLES, ARPACK, or yaml-cpp from source ###
+# SPOOLES include flags (e.g. -I$(HOME)/SPOOLES.2.2 )
+SPOOLES_INCLUDE   = -I/usr/include/spooles/
+# SPOOLES library flags (e.g. $(HOME)/SPOOLES.2.2/spooles.a)
+SPOOLES_LIBS      = -lspooles
+#
+# ARPACK include flags (e.g. -I$(HOME)/ARPACK)
+ARPACK_INCLUDE    =
+# ARPACK library flags (e.g. $(HOME)/ARPACK/libarpack_INTEL.a)
+ARPACK_LIBS       = $(HOME)/ARPACK/libarpack_INTEL.a 
+#ARPACK_LIBS       = -larpack -llapack -lblas
+#
+# yaml-cpp include flags (e.g. -I$(HOME)/yaml-cpp/include)
+YAML_INCLUDE      = -I/usr/include/
+# yaml-cpp library flags (e.g. -L$(HOME)/yaml-cpp/build -lyaml-cpp)
+YAML_LIBS         = -lyaml-cpp
+
+# Get the CFLAGS and LIBS from pkg-config (preCICE version >= 1.4.0).
+# If pkg-config cannot find the libprecice.pc meta-file, you may need to set the
+# path where this is stored into PKG_CONFIG_PATH when building the adapter.
+PKGCONF_CFLAGS  = $(shell pkg-config --cflags libprecice)
+PKGCONF_LIBS    = $(shell pkg-config --libs libprecice)
+
+# Specify where to store the generated .o files
+OBJDIR = bin
+
+# To get the correct path, export the required folders for pkg-config. Example: export PKG_CONFIG_PATH=~/PaStiX/pastix_i8/lib/pkgconfig/:~/PaStiX/hwloc_i8/lib/pkgconfig/:~/PaStiX/parsec_i8/lib/pkgconfig/
+
+PASTIX_INCLUDE = $(shell pkg-config --cflags pastix)
+PASTIX_LIBS = $(shell pkg-config --libs pastix hwloc parsec lapacke) $(HOME)/OpenBLAS_i8/libopenblas.a \
```

Comment:
Is this really `lapacke`? Feels like a typo.

Reply:
Apparently `lapacke` is a C wrapper of `lapack` (written in FORTRAN) or something like that. But now that you say it, I tried `lapack` and it worked too... Because none of them were required.
I think the required dependency is contained in `OpenBLAS_i8/libopenblas.a`. So let's just remove it alltogether